### ISSUE TITLE
[Feature Request] fvm's own messages should output to stderr

### ISSUE THREAD

**Issue by ened** [leoafarias/fvm#178]
**Is your feature request related to a problem? Please describe.**

A example of release scripts we are running is doing this:

```
fvm flutter --version > assets/json/flutter-version.json
```

This now results in a output of:

```
FVM: Running version 1.20.4
Flutter 1.20.4 • channel unknown • unknown source
Framework • revision fba99f6cf9 (6 weeks ago) • 2020-09-14 15:32:52 -0700
Engine • revision d1bc06f032
Tools • Dart 2.9.2
```

The first line contains FVM's _own_ output.

**Describe the solution you'd like**

1. `fvm`'s own output should by default go to stderr 
OR
1. `fvm` should support a new option `-q` which turns of any output, unless it's a error

**Comment by leoafarias** [comment]
@ened Could you send me a sample of that `flutter-version.json` file to see what you are trying to do?

**Comment by leoafarias** [comment]
I think I understand it though. Moving the `FVM: Running version ...` to another log level

**Comment by leoafarias** [comment]
Fixed on version 1.3.6
